docs/Doxyfile.in

```diff
@@ -764,7 +764,7 @@ WARN_LOGFILE           =
 # spaces.
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = @CMAKE_CURRENT_SOURCE_DIR@/src/ @CMAKE_CURRENT_SOURCE_DIR@/include/
+INPUT                  = @CMAKE_CURRENT_SOURCE_DIR@/src/ @CMAKE_CURRENT_SOURCE_DIR@/include/ @CMAKE_CURRENT_SOURCE_DIR@/docs/
```

Comment:
Why do we need to add this one?

Reply:
To make doxygen aware of the introductory articles (like [this](https://vinecopulib.github.io/vinecopulib/overview-bicop.html)), which are located in `docs/`.